From fb2355adb3ce4de72d36830f608fb188c18d1981 Mon Sep 17 00:00:00 2001
From: Ed Santiago <santiago@redhat.com>
Date: Fri, 1 Oct 2021 05:23:16 -0600
Subject: [PATCH] Gating tests: fix permissions error

...in volume test. Looks like Bodhi gating tests run from a
nonwritable directory. I feel really stupid for not realizing
this when I first tried to fix this bug two weeks ago.

Signed-off-by: Ed Santiago <santiago@redhat.com>
---
 test/system/160-volumes.bats | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/system/160-volumes.bats b/test/system/160-volumes.bats
index b48198a1899..c02525e0d3b 100644
--- a/test/system/160-volumes.bats
+++ b/test/system/160-volumes.bats
@@ -191,7 +191,7 @@ EOF
     run_podman run --rm -v my_vol:/data $IMAGE sh -c "echo hello >> /data/test"
     run_podman volume create my_vol2
 
-    tarfile=hello$(random_string | tr A-Z a-z).tar
+    tarfile=${PODMAN_TMPDIR}/hello$(random_string | tr A-Z a-z).tar
     run_podman volume export my_vol --output=$tarfile
     # we want to use `run_podman volume export my_vol` but run_podman is wrapping EOF
     run_podman volume import my_vol2 - < $tarfile
